client/state/selectors/get-site-timezone-string.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Return the site setting `timezone_string` value from the state-tree
+ *
+ * @param  {Object}  state - Global state tree
+ * @param  {Number}  siteId - Site ID
+ * @return {?String} site setting timezone
+ */
+export default function getSiteTimezoneString( state, siteId ) {
```

Comment:
Do we standardize this on the backend? In our current UI we do something strange like offer region tz names eg `America/Araguaina` along with `UTC+3` options in a single select dropdown.